srv/salt/_modules/cephdisks.py

```diff
@@ -512,6 +512,35 @@ def list_(**kwargs):
     return hwd.assemble_device_list()
 
 
+def disable_write_cache(disk="All", persistent=False, **kwargs):
+    """
+    Disable write cache (WCE) on rotational disks
+    """
+    hwd = HardwareDetections(**kwargs)
+    sdparm_path = hwd._which('sdparm')
+
+
+    if disk == "All":
```

Comment:
What about inverting the logic. If the user provides no `disk` parameter, it's assumed to auto-select all disks. If there is a disk parameter set the user only wants to change the settings for one.

Also hardcoded it to (capital)'All' which will be hard to guess.

A couple of more considerations:

* If someone disabled it, he maybe also wants to enable it again.
* If we have a disable and a enable function, we may also need a help function

Taking these two facts into account, it might even be wise to move that to it's own module where you can make use of a `xxxxx.help` notation.

Reply:
> What about inverting the logic. If the user provides no disk parameter, it's assumed to auto-select all disks. If there is a disk parameter set the user only wants to change the settings for one.

Isn't that already the case with the default `disk="All"`?

I do agree with your criticism of "All", something like `if disks.lower() == "all":` would guard against wrongly capitalized input.

> * If someone disabled it, he maybe also wants to enable it again.

Sounds reasonable.

> * If we have a disable and a enable function, we may also need a help function

I also agree with this. As of now, there is only `cephdisks.list`, could we use `cephdisks.help` to cover `list`, `disable_write_cache` and `enable_write_cache` (plus maybe a mention of `version`)? Moving the dis-/enablement of the write cache to another module might not be needed.